app/src/main/java/com/rokt/roktdemo/ui/layouts/DemoLayoutsPage.kt

```diff
@@ -83,6 +84,7 @@ fun DemoLayoutsPage(
 private fun ScannerContent(
     backPressed: () -> Unit,
     viewModel: ScanQrViewModel,
+    mainActivityViewModel: MainActivityViewModel,
```

Comment:
Ideally viewModel should not pass, use callbacks or events to call ViewModel

Reply:
Thanks, refactored.